app/connectors/connectors/http4/src/main/resources/META-INF/syndesis/connector/http4.json

```diff
@@ -0,0 +1,167 @@
+{
+      "id": "http4",
+      "name": "HTTP4",
+      "icon" : "http4",
+      "description": "Http4 Connector",
+      "dependencies": [
+          {
+              "type": "MAVEN",
+              "id": "@project.groupId@:@project.artifactId@:@project.version@"
+          }
+      ],
+      "tags": [
+        "verifier"
+      ],
+      "properties": {
+        "hostname": {
+          "kind": "path",
+          "displayName": "Hostname",
+          "group": "common",
+          "required": true,
+          "type": "string",
+          "javaType": "java.lang.String",
+          "deprecated": false,
+          "secret": false,
+          "description": "Server Hostname"
+        },

```

Comment:
I'm not sure if we want to set hostname & co as global properties or if it may be better to set them at action level, @kcbabo what do you think ?